backend/api/views/docclass.py

```diff
@@ -26,28 +27,63 @@ def get_document_class_by_id(id):
 @docclass.route("/document_class/new", methods=["POST"])
 def add_document_class():
     """ function that is called when you visit /document_class/new, creates a new docclass """
-    data = request.get_json()
-    logger.info(data)
+    data = request.form
```

Comment:
This PR changes the body through which parameters are passed to PUT and POST requests from `json` to `FormData`. Any request that handles files requires that we use FormData; however using `request.json` or `request.get_json()` on these does not work. For consistency's sake, and because we can get the proper results with FormData provided we changed the model `__init__()`s as above, I changed any instance of `request.json` or `request.get_json()` to `request.form`.